cime_config/pop_cime_py/__init__.py

```diff
@@ -0,0 +1 @@
+from .buildnml import buildnml
```

Comment:
Is this required for it to work or just an optional thing that you like to do?

Reply:
As written, it allows users to call the `buildnml()` function inside the `buildnml` module with

```py
import pop_cime_py
pop_cime_py.buildnml()
```

If we had an empty `__init__.py`, users would instead need

```
import pop_cime_py
pop_cime_py.buildnml.buildnml()
```

So it's required for the [following CIME code](https://github.com/ESMCI/cime/blob/04d76ee75c0b9e269c0032640d27b2fbeea9007f/scripts/lib/CIME/utils.py#L431) to work

```py
mod = importlib.import_module(f"{compname}_cime_py")
getattr(mod, subname)(*subargs)
```

and without it, we'd have to do something clunkier like

```diff
mod = importlib.import_module(f"{compname}_cime_py")
-getattr(mod, subname)(*subargs)
+getattr(getattr(mod, subname),subname)(*subargs)
```

Reply:
That makes sense. Thanks for the explanation!